src/Microsoft.AspNetCore.SignalR.Common/IHubProxy.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Channels;
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public interface IHubProxy
+    {
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="methodName"></param>
+        /// <param name="args"></param>
+        Task Invoke(string methodName, params object[] args);
```

Comment:
And if we're going to fix this naming issue, now's our best opportunity to do so. It's not like we can't revert it after getting feedback.